### PR TITLE
Attempt to send test results from monitoring to Cypress Dashboard

### PR DESCRIPTION
**Pull request by wdsrocha**
This PR has the objective to alert when some monitoring test fails on our selected Slack channels, instead of sending to the default Slack channel where every other test goes.

One way to do this is to send the tests to Cypress Dashboard, which automatically sends any failures to the Slack channel. Given that the monitoring checkout-ui-tests container already has the Cypress Dashboard [project id](https://github.com/vtex/checkout-ui-tests/blob/master/src/monitoring/index.js#L89) and [access key](https://github.com/vtex/checkout-ui-tests/blob/master/cypress.json#L10) (or is the access key [here](https://github.com/vtex/checkout-ui-tests/blob/9034681f36d6500774e0ba414f4ddcb6d0aca1fb/package.json#L55-L57)?), all I think needs to be done is to pass the `key` and `recording` parameters to the `index.js` file configuration.

Btw, yes, the key shouldn't be explicit and I think we should fix this soon

## How to manually test this

I'm not sure, as we still don't know how the tests are executed in VTEX infrastructure. I've heard that all that infra does is put a cron on a docker container, so I suppose that running `yarn test:ci-stable` will simulate the same thing that happens in the infra, as this command run the docker container as well.

In short, just clone this repository and run `yarn test:ci-stable`. Open the https://dashboard.cypress.io/projects/kobqo4/runs and check for the results.

### CODE REVIEW

**Review by darlenedms — CHANGES_REQUESTED** [review]
Missing PR description and the CHANGELOG 😛

**Comment by wdsrocha** [comment]
I still want to do more tests, so I'll close this for now.

**Comment by wdsrocha** [comment]
oof I can't reopen this PR due [this KI](https://github.com/isaacs/github/issues/361). Created another one https://github.com/vtex/checkout-ui-tests/pull/99
